src/main/java/org/spongepowered/common/mixin/core/entity/EntityLivingBaseMixin.java

```diff
@@ -1170,4 +1171,16 @@ private void onResetActiveHand(final CallbackInfo ci) {
 
     // End implementation of UseItemStackEvent
 
+    @Redirect(method = "onLivingUpdate", at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/EntityLivingBase;collideWithNearbyEntities()V"))
+    private void collisions$runCollisions(EntityLivingBase self) {
```

Comment:
If this is supposed to be added when the collision plugin has enabled collisions, this should be in a separate mixin that gets applied.

Reply:
This is necessary to enter the `EntityCollisionState` and it should always be applied. 
The name was probably copied from the old Mixin.

Reply:
I've renamed it to `impl$runCollisions`.